.stylelintrc

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
Some rules have been renamed or deprecated, best run Stylelint and look out for any warnings.